rewriter/flatten.cc

```diff
@@ -276,7 +294,7 @@ class FlattenWalk {
         // that we don't know the 'staticness level' of a sig, as it depends on the method that follows it (whether that
         // method has a `self.` or not), so we'll fill that information in later
         if (send->fun == core::Names::sig()) {
-            curMethodSet().pushScope(0);
+            curMethodSet().pushScope(0, false);
```

Comment:
One of two suggestions:

- assign the literal `false` to a local before passing it around (i.e., `auto isClassMethod = false; go(isClassMethod);`)
- create an `enum class` instead of using booleans. See for example:

https://github.com/sorbet/sorbet/blob/7e4349d359e66dc6512b138778e2b609c4425b96/rewriter/OpusEnum.cc#L16-L19

https://github.com/sorbet/sorbet/blob/7e4349d359e66dc6512b138778e2b609c4425b96/rewriter/OpusEnum.cc#L193

Passing around booleans unnamed is not fun in the long run.

Reply:
Replaced with an enum!